### PR TITLE
Feature/eth chain bridge

### PR DESCRIPTION
**Pull request by evgeny-s**

### CODE REVIEW

File: frame/chainbridge/src/lib.rs [hunk 1]
```diff
@@ -618,4 +618,8 @@ impl<T: Trait> EnsureOrigin<T::Origin> for EnsureBridge<T> {
             r => Err(T::Origin::from(r)),
         })
     }
+
```
Comment (evgeny-s) [inline frame/chainbridge/src/lib.rs:621]: Shouldn't we add this before the method:
```
#[cfg(feature = "runtime-benchmarks")]
```

File: Dockerfile.tests [hunk 2]
```diff
@@ -11,4 +11,4 @@ RUN curl https://sh.rustup.rs -sSf | sh -s -- -y && \
 	export PATH=$PATH:$HOME/.cargo/bin && \
 	scripts/init.sh && \
 	TRYBUILD=overwrite cargo test --workspace --locked --release --verbose --features runtime-benchmarks --manifest-path bin/node/cli/Cargo.toml && \
-	WASM_BUILD_NO_COLOR=1 SUBSTRATE_TEST_TIMEOUT=1 cargo test -p substrate-test-utils --release --verbose --locked -- --ignored timeout
+	WASM_BUILD_NO_COLOR=1 SUBSTRATE_TEST_TIMEOUT=1 cargo test -p substrate-test-utils --release --verbose --locked -- --ignored timeout
```
Comment (evgeny-s) [inline Dockerfile.tests]: Why do we change this line? Please revert changes

File: frame/chainbridge/src/lib.rs (hunk 1 above)
Comment (huythanhcere) [reply frame/chainbridge/src/lib.rs:621]: I added in the latest commit.

File: Dockerfile.tests (hunk 2 above)
Comment (huythanhcere) [reply Dockerfile.tests]: I reverted in the latest commit.

File: frame/chainbridge/src/lib.rs (hunk 1 above)
Comment (evgeny-s) [reply frame/chainbridge/src/lib.rs:621]: Remove space. It should be the same padding for function and annotation
Screenshot from 2021-04-07 14-48-26

File: bin/node/runtime/Cargo.toml [hunk 3]
```diff
@@ -176,6 +176,7 @@ runtime-benchmarks = [
 	"pallet-treasury/runtime-benchmarks",
 	"pallet-utility/runtime-benchmarks",
 	"pallet-vesting/runtime-benchmarks",
+	"pallet-chainbridge/runtime-benchmarks",
```
Comment (evgeny-s) [inline bin/node/runtime/Cargo.toml:179]: Why do we need this line here? Revert plz

Comment (huythanhcere) [reply bin/node/runtime/Cargo.toml:179]: Without this build will success but test will fail. @evgeny-s

File: frame/chainbridge/src/lib.rs (hunk 1 above)
Comment (huythanhcere) [reply frame/chainbridge/src/lib.rs:621]: So strange, in my VS code it is same. I will fix its.

**Pull request merged**